src/features/orientation/infrastructure/presentation/components/address-field/address-field.component.html

```diff
@@ -4,6 +4,7 @@
     #addressInput
     type="search"
     placeholder="Entrez une adresse"
+    [(ngModel)]="addressFromCoordiantes"
```

Comment:
```suggestion
```
Il y a deux manières de gérer les formulaires avec Angular :
- [Template-driven form](https://angular.io/guide/forms)
- [Reactive forms](https://angular.io/guide/reactive-forms)
`ngModel` fait partie des Template-driven forms, comme on essaye d'utiliser au maximum les observables, on ne va utiliser que des Reactive forms.
Pour mettre une valeur dans un input d'un reactive form, il faut changer la valeur dans le form comtrol correspondant, par exemple : `this.orientationLayout.filterForm.get('address')?.setValue('77 avenue de Ségur 75007, Paris);`